package.json

```diff
@@ -1,42 +1,47 @@
-{
-  "name": "moramia-website",
-  "version": "1.0.0",
-  "description": "Moramia website",
-  "scripts": {
-    "build": "webpack && node prepare.js",
-    "start": "npm run build && webpack-dev-server --mode development"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/pushdev-code/moramia-website.git"
-  },
-  "keywords": [
-    "moramia"
-  ],
-  "author": "pushdev",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/pushdev-code/moramia-website/issues"
-  },
-  "homepage": "https://github.com/pushdev-code/moramia-website#readme",
-  "dependencies": {
-    "ncp": "^2.0.0"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.9.6",
-    "@babel/preset-env": "^7.9.6",
-    "babel-loader": "^8.1.0",
-    "css-loader": "^3.5.3",
-    "file-loader": "^6.0.0",
-    "html-loader": "^1.1.0",
-    "html-webpack-plugin": "^4.3.0",
-    "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "^4.14.1",
-    "node-static": "^0.7.11",
-    "sass-loader": "^8.0.2",
-    "style-loader": "^1.2.1",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.11.0"
-  }
-}
+{
+  "name": "moramia-website",
+  "version": "1.0.0",
+  "description": "Moramia website",
+  "scripts": {
+    "build": "webpack && node prepare.js",
+    "serve": "browser-refresh ./src/server.js",
+    "start": "node ./src/server.js",
+    "start-development": "npm run build && webpack-dev-server --mode development"

```

Comment:
make sure these tasks are included in the ReadMe so any developer can take a look and setup the environment with just using that file's information.